src/app/api/crud/groups.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (C) 2021, Pyronear contributors.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+
+from typing import List, Dict, Mapping, Any
+from sqlalchemy import Table
+from fastapi import HTTPException, status
+
+from app.api import crud
+from app.api.schemas import AccessType
+from app.db import (
+    accesses, users, groups,
+    sites, events, devices, media,
+    installations, alerts)
+
+
+async def get_entity_group_id(table: Table, entry_id: int) -> int:
```

Comment:
Improvement suggestion: for cleaner signature, I suggest passing directly the entry. Then the ifs can be checking whether it is `AccessRead` or other reading schemas

Reply:
Not sure it will be more efficient. One will need to retrieve the entity before doing the group_id check

Reply:
But it's true that it is already what we do.
Ok I'll try. But I fear that the Schema checks might be a more brittle design.